### PR TITLE
Translated views and devise messages to Ukrainian

### PR DESCRIPTION
**Pull request by simplews**
- [ ] @aadmin 
- [ ] @arsenlol 
- [ ] @Kmalynss 

* Translated views to Ukrainian
* Added Ukrainian localisation file for devise

Also gem 'factory_girl_rails' moved from group :development to main, added link to heroku in Readme

### CODE REVIEW

**Comment by simplews** [comment]
Rspec tests was successful at local copy but fails on Travis. Have no idea what to do with it

File: app/models/issue.rb [hunk 1]
```diff
@@ -8,13 +9,13 @@ class Issue < ApplicationRecord
             presence: true
   validates :name, length: { maximum: 255 },
                    format: { with: REGEXP_NAME,
-                             message: 'should contain only letters' }
+                             message: 'Повинно містити лише літери' }
   validates :phone, length: { maximum: 50 },
                     format: { with: REGEXP_PHONE,
-                              message: 'should contain only numbers' }
+                              message: 'Повинно містити лише цифри' }
```
Comment (f1nwe) [inline app/models/issue.rb]: Maybe specify where is a problem?
'Імʼя повинне містити тільки літери'
"Номер телефону повинен містити лише літери"

File: app/views/issues/new.html.slim [hunk 2]
```diff
@@ -1,20 +1,20 @@
-h1 Create new issue
-= link_to 'Home', root_path
+h1 Відправити нове звернення
```
Comment (f1nwe) [inline app/views/issues/new.html.slim]: "Надіслати" not "відправити"

File: app/views/issues/new.html.slim [hunk 3]
```diff
@@ -1,20 +1,20 @@
...
-        = f.input :name, label: 'Name',
-                    placeholder: 'Vasya Pupkin'
-        = f.input :address, label: 'Address',
-                    placeholder: 'Khotkevycha str. 33/3'
-        = f.input :phone, label: 'Phone',
+        = f.input :name, label: 'Ім\'я',
+                    placeholder: 'Петро Петрович Петренко'
+        = f.input :address, label: 'Адреса',
+                    placeholder: 'вул. Грушевського 21'
+        = f.input :phone, label: 'Телефон',
```
Comment (f1nwe) [inline app/views/issues/new.html.slim]: "Номер телефону"

File: app/views/layouts/_logged_content.html.slim [hunk 4]
```diff
@@ -1,11 +1,11 @@
...
   - if current_user.admin? || current_user.moderator?
       li.nav-text-link
-        = link_to 'All users', account_admin_users_path
+        = link_to 'Всі користувачі', account_admin_users_path
       li.nav-text-link
-        = link_to 'All issues', account_admin_issues_path
+        = link_to 'Всі звернення', account_admin_issues_path
   li
-    = link_to 'Report issue', new_issue_path,
+    = link_to 'Відправити звернення', new_issue_path,
```
Comment (f1nwe) [inline app/views/layouts/_logged_content.html.slim]: "Надіслати звернення"

**Comment by f1nwe** [comment]
http://stackoverflow.com/questions/11620767/devise-translation-missing-error-with-subclassed-controller

Here is answer to you question.
You must add uk localisation to devise gem.

To pass tests you must add somthig as

```
uk:
  devise:
    omniauth_callbacks:
          failure: "Could not authenticate you from %{kind} because \"%{reason}\"."
          success: "Вхід з облікового запису %{kind} успішний."
```
in config/locales/devise.uk.yml

**Comment by f1nwe** [comment]
Also please squash commits as Kostya said to us.

**Comment by simplews** [comment]
Yes, later I found that file config/locales/devise.en.yml wasn't added to commit, so it was fixed.

File: config/locales/devise.uk.yml [hunk 5]
```diff
@@ -0,0 +1,63 @@
+#Український переклад для https://github.com/plataformatec/devise/tree/v4.2.0
+#Інші переклади на http://github.com/plataformatec/devise/wiki/I18n
+
+uk:
+  devise:
+    confirmations:
+      confirmed: "Ваш обліковий запис було успішно підтверджено."
+      send_instructions: "На Ваш email надіслано інструкції з підтвердження облікового запису."
+      send_paranoid_instructions: "Якщо Ваш email існує у нашій базі даних, Вам надіслано інструкції з підтвердження облікового запису."
```
Comment (f1nwe) [inline config/locales/devise.uk.yml]: This sounds little wierd.

File: config/locales/devise.uk.yml [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+      confirmation_instructions:
+        subject: "Інструкції з підтвердження облікового запису."
+      reset_password_instructions:
+        subject: "Інструкції з відновлення пароля."
+      unlock_instructions:
+        subject: "Інструкції з розблокування облікового запису"
+      password_change:
+        subject: "Пароль був змінений."
+    omniauth_callbacks:
+      failure: "Неможливо увійти з облікового запису %{kind} через те, що обліковий запис \"%{reason}\"."
```
Comment (f1nwe) [inline config/locales/devise.uk.yml]: There are tautology in this sentence

File: config/locales/devise.uk.yml [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+      already_signed_out: "Вихід успішний."
+    unlocks:
+      send_instructions: "Протягом декількох хвилин на Ваш email буде надіслано інструкції з розблокування облікового запису."
+      send_paranoid_instructions: "Якщо ваш обліковий запис існує, то ви отримаєте повідомлення електронної пошти з інструкціями, як розблокувати його протягом декількох хвилин."
+      unlocked: "Ваш обліковий запис було успішно розблоковано."
+  errors:
+    messages:
+      already_confirmed: "вже було підтверджено, спробуйте увійти"
+      confirmation_period_expired: "повинен бути підтверджений протягом періоду %{period}, будь ласка, запитайте підтвердження ще раз"
+      expired: "прострочено, створіть новий"
```
Comment (f1nwe) [inline config/locales/devise.uk.yml]: "Строк" is a russian word. Use "Протерміновано"

File: config/locales/devise.uk.yml (hunk 6 above)
Comment (simplews) [inline config/locales/devise.uk.yml]: fixed

File: config/locales/devise.uk.yml (hunk 5 above)
Comment (simplews) [inline config/locales/devise.uk.yml]: fixed

**Pull request merged**